drivers/usb/common/nrf_usbd_common/nrf_usbd_common.c

```diff
@@ -1687,6 +1706,9 @@ void nrf_usbd_common_disable(void)
 	/* Stop just in case */
 	nrf_usbd_common_stop();
 
+	/* Make sure DMA is not active */
+	k_sem_take(&dma_available, K_FOREVER);
```

Comment:
I messed up during the rebases (when doing prefix change) and the code isn't really matching the commit message where I wrote `Mark nrf_usbd_common_stop() as static because the caller must acquire DMA semaphore before calling this function and the only place where it is used is already acquiring the semaphore.`. I will change the code to match the comment - i.e. take semaphore before calling `nrf_usbd_common_stop()`.